src/client/components/collection/Collection.js

```diff
@@ -25,31 +24,31 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function Collection() {
-  // todo: make dynamic
-  // const { id } = useParams();
+  const { id } = useParams();
+
+  const collection = useCollection(id);
 
   const {
-    current: {
-      id,
-      comments,
-      description,
-      image = {},
-      subitems: items,
-      voteScore: likes,
-      name,
-      // todo: get similar collections in same call
-      // similarCollections,
-      views,
-      members,
-    },
-  } = useContext(CollectionContext);
+    comments,
+    description,
+    image = {},
+    subitems: items,
+    voteScore: likes,
+    name,
+    // todo: get similar collections in same call
+    // similarCollections,
+    views,
+    members,
+    loadingCollection,
+  } = collection;
 
   const { pictureId } = image;
   const creator = members.find(({ type }) => type === MEMBER_TYPES.OWNER);
   const contributors = members.filter(({ id: mId }) => mId !== creator.id);
-  const imageUrl =
-    buildImageUrl({ id, pictureId, quality: DEFAULT_PICTURE_QUALITY }) ||
-    ITEM_DEFAULT_IMAGE;
+  const imageUrl = loadingCollection
+    ? image.thumbnailUrl
```

Comment:
It is better to use `buildImageUrl` instead of `thumbnailUrl` because the fonction can return a higher quality image.

Reply:
Hey Kim, we are still using the `buildImageUrl` this is only for the placeholder image